src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -444,7 +444,12 @@ unique_ptr<CatalogEntry> TableCatalogEntry::SetDefault(ClientContext &context, S
 		auto copy = columns[i].Copy();
 		if (default_idx == i) {
 			// set the default value of this column
-			D_ASSERT(!copy.Generated()); // Shouldnt reach here - DEFAULT value isn't supported for Generated Columns
```

Comment:
turns out it actually reaches here - this happens when you create a generated column and try adding a default value through ALTER VALUE, so I just changed the assertion with a more elegant exception